__tests__/app.test.js

```diff
@@ -138,20 +138,83 @@ describe("GET /api/articles", () => {
             })
           });
     });
-    test("status:400, responds with an error message when passed an invalid sort_by", () => {
+    test("status:400, sends an appropriate status and error message when passed an invalid sort_by", () => {
       return request(app)
         .get("/api/articles?sort_by=not_a_sort_by")
         .expect(400)
         .then(({ body }) => {
           expect(body.msg).toBe("Bad request");
         });
     });
-    test("status:400, responds with an error message when passed an invalid order_by", () => {
+    test("status:400, sends an appropriate status and error message when passed an invalid order_by", () => {
       return request(app)
         .get("/api/articles?order_by=random_order")
         .expect(400)
         .then(({ body }) => {
           expect(body.msg).toBe("Bad request");
         });
     });
+})
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status:200, responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),  
+            author: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
You're absolutely not expecting any article_id here. You specifically requested the data for article 1. If I was served comments from article 12 I'd be baffled and this would pass,